ceph-releases/luminous/ubuntu/16.04/daemon/disk_introspection.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+HOSTNAME=$(uname -n)
+MY_NAMESPACE=$(kubectl get pods --all-namespaces -o jsonpath="{.items[?(@.metadata.name == \"${HOSTNAME}\")].metadata.namespace}")
```

Comment:
@font this actually does not seem to work:

`error: error executing jsonpath "{.items[?(@.metadata.name == shell-demo)].metadata.namespace}": unrecognized identifier shell-demo`.

I know this might go away at some point and triggering the API is better but what about `/var/run/secrets/kubernetes.io/serviceaccount/namespace`?